sf-stream/src/protos/extractor.proto

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+message Transaction {
+  uint64 timestamp = 1;
```

Comment:
https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp should probably be used here. It's a well-known type that I think all Protobuf language supported must have support of, so it usually creates better pattern in generated code.